src/PlaywrightSharp.Chromium/Protocol/RemoteObject.cs

```diff
@@ -0,0 +1,6 @@
+namespace PlaywrightSharp.Chromium.Protocol.Runtime
+{
+    internal partial class RemoteObject : IRemoteObject

```

Comment:
all generated types are `partial` and implementing playwright's interfaces could be done this way (in case it has a bit of logic like firefox)